act-framework.bs

```diff
@@ -159,8 +159,17 @@ Editor note: This section describes the requirements of tests that have to be cr
 Accuracy Benchmarking {#quality-accuracy}
 -----------------------------------------
 
-Editor note: This section describes how to measure the rate of incorrect results to correct results of a rule. Measuring this accuracy, not just on test data, but on pages and components that the rules would actually be applied to, is important to give users of the test results confidence in the accuracy of this data.
+The web is ever changing, and technologies are used in such diverse and creative ways that it is impossible to predict in advance, all the ways that accessibility issues can occur and all the ways they can be solved for. When writing ACT Rules, it is almost inevitable that exceptions will be overlooked during the design of a rule, or that new technologies will emerge that introduce new exceptions.
 
+This makes it important to be able to regularly test if the rule has the accuracy that is expected of it. This can be done by benchmark testing. In benchmark testing, the accuracy of a rule is measured, by comparing it's results to those obtained through accessibility expert testing.
+
+There are several ways this can be done. For instance, accessibility test tools can implement a feature which lets users indicate that that a result is in error, or pages that for which accessibility results are known, can be tested using ATT, and the results are compared.
+
+The accuracy of a rule is the average between the false positives and false positives, which are in tern calculated as follows:
```

Comment:
Hi Wilco, After today's conversation, I thought of a real life example. In support of WCAG Sufficient Technique [H44](https://www.w3.org/TR/WCAG20-TECHS/H44.html), we had a rule to check for form labels. However, because at the time of writing the rule, some screen readers did not support implicit labels, see [UA note](https://www.w3.org/WAI/WCAG20/Techniques/ua-notes/html#H44), the rule would only pass if explicit labels were used. Recently, we received feedback from our development teams that implicit labels, wrapping input elements inside a <label> element, works just fine and are read by the screen reader. We have updated our rule to support both implicit and explicit labels while is supported by both HTML and XHTML specifications to associate text labels with form controls and meets the requirements of the related WCAG SC.